docs/static/monitoring-apis.asciidoc

```diff
@@ -201,6 +314,117 @@ Example response:
   }
 --------------------------------------------------
 
+==== Mem Stats
+
+The following request returns a JSON document containing memory stats: 
+
+[source,js]
+--------------------------------------------------
+GET /_node/stats/mem
+--------------------------------------------------
+
+Example response:
+
+[source,js]
+--------------------------------------------------
+{
+  "mem": {
+    "heap_used_in_bytes": 262641008,
+    "heap_used_percent": 12,
+    "heap_committed_in_bytes": 519045120,
+    "heap_max_in_bytes": 2075918336,
+    "non_heap_used_in_bytes": 184060512,
+    "non_heap_committed_in_bytes": 195870720,
+    "pools": {
+      "survivor": {
+        "peak_used_in_bytes": 8912896,
+        "used_in_bytes": 9280256,
+        "peak_max_in_bytes": 35782656,
+        "max_in_bytes": 71565312,
+        "committed_in_bytes": 17825792
+      },
+      "old": {
+        "peak_used_in_bytes": 112649208,
+        "used_in_bytes": 170055984,
+        "peak_max_in_bytes": 715849728,
+        "max_in_bytes": 1431699456,
+        "committed_in_bytes": 357957632
+      },
+      "young": {
+        "peak_used_in_bytes": 71630848,
+        "used_in_bytes": 83304768,
+        "peak_max_in_bytes": 286326784,
+        "max_in_bytes": 572653568,
+        "committed_in_bytes": 143261696
+      }
+    }
+  }
+--------------------------------------------------
+
+==== Pipeline Stats
+
+The following request returns a JSON document containing pipeline stats, including the number of events that were
+input, filtered, or output by the pipeline. The request also returns stats for each configured input, filter, or
+output stage. coming[5.0.0-alpha4,Stats for input stages are not yet available]
+
+[source,js]
+--------------------------------------------------
+GET /_node/stats/pipeline
+--------------------------------------------------
+
+//REVIEWERS: Note that when you call /_node/stats, it does not return the pipeline stats. You need to specify /_node/stats/pipeline specifically. This makes me wonder if it shouldn't be its own API topic. WDYT?
```

Comment:
This is intentional. `/_node/stats/pipeline` is too verbose (customers have complex plugin configs) to be included in ` /_node/stats`. Yes it should be a separate API topic.